src/main/java/com/es/phoneshop/model/validator/parameterValidator/NotBlankParameterValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.es.phoneshop.model.validator.parameterValidator;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Map;
+
+public class NotBlankParameterValidator implements ParameterValidator {
```

Comment:
not good enough to create a name based on you implementation details
now you're checking only isBlank(), but if in future you need a check for maxLength, then the validator will be renamed too
some general name would be better